esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3488,10 +3504,17 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     ForEach(*it)
         addToLogicalFileList(it->query(), NULL, version, logicalFiles);
 
-    if (version >= 1.24)
-        resp.setCacheHint(cacheHint);
+    if ((oldCacheHint != cacheHint) && (totalFiles > maxFiles))
```

Comment:
The user will not see the warning when switching to the 2nd page. Not sure we should keep poping-up the same warning from page to page.